lib/core/_general.scss

```diff
@@ -0,0 +1,23 @@
+@import 'helpers';
+
+
+
+@if ($hill-layout-css-general) {
+    @for $index from 1 through $hill-layout-space-multiplier {
+        @if $index > 1 {
+            $multiplier: "-#{$index}x";
+        } @else {
+            $multiplier: '';
+        }
+
+        [#{$hill-layout-attribute-name}*="space-top#{$multiplier}"]       { margin-top:    #{$hill-layout-space * $index}; }
+        [#{$hill-layout-attribute-name}*="space-right#{$multiplier}"]     { margin-right:  #{$hill-layout-space * $index}; }
+        [#{$hill-layout-attribute-name}*="space-bottom#{$multiplier}"]    { margin-bottom: #{$hill-layout-space * $index}; }
+        [#{$hill-layout-attribute-name}*="space-left#{$multiplier}"]      { margin-left:   #{$hill-layout-space * $index}; }
+    }
+
+    [#{$hill-layout-attribute-name}*="float-left"]     { @extend  %_floatLeft; }
```

Comment:
Sometimes you need only floating in your layout structure. But i think it is very exotic and can be removed from hillcss. What do you think?